src/frontend/src/pages/HomePage/Home.tsx

```diff
@@ -7,16 +7,19 @@ import { useCurrentUser } from '../../hooks/users.hooks';
 import { isGuest } from 'shared';
 import GuestHomePage from './GuestHomePage';
 import MemberHomePage from './MemberHomePage';
-import { useState } from 'react';
+import PNMHomePage from './PNMHomePage';
+import { useHomePageContext } from '../../app/HomePageContext';
 
 const Home = () => {
   const user = useCurrentUser();
-  const [onMemberHomePage, setOnMemberHomePage] = useState(false);
-  return isGuest(user.role) && !onMemberHomePage ? (
-    <GuestHomePage user={user} setOnMemberHomePage={setOnMemberHomePage} />
-  ) : (
-    <MemberHomePage user={user} />
-  );
+  const { onGuestHomePage, setOnGuestHomePage, onPNMHomePage, setOnPNMHomePage } = useHomePageContext();
```

Comment:
I feel like our lives would be significantly simplified if we made a different url for the guest home page and the pnm home page im ngl lol